tasks/installation.yml

```diff
@@ -0,0 +1,8 @@
+---
+- name: install firewalld related packages
+  package:
+    name: '{{ firewalld_packages }}'
+    state: present
+  tags:
```

Comment:
tags can be removed in `installation.yml` as they are already specified in `tasks/main.yml` on include